api/v1_users_coins.go

```diff
@@ -89,18 +89,41 @@ func (app *ApiServer) v1UsersCoins(c *fiber.Ctx) error {
 			FROM balances
 			GROUP BY balances.mint
 		),
+		-- Include owned coins even if they have zero balance
+		owned_coins AS (
```

Comment:
The extra CTE expressions seem a little unnecessary / overcomplicated?
I would think we just want to join in artist coins to get the info and then we can use a `where artist_coins.user_id = @userId OR balance > 0` to filter the final set of rows.